pkg/conversions/conversions.go

```diff
@@ -18,10 +18,6 @@ func GibToMib(gib int64) int64 {
 	return gib * int64(units.KiB)
 }
 
-func MibToGiB(mib int64) int64 {
```

Comment:
these helper funcs are actually nice to have. maybe worth keeping them.

Reply:
Done